### ISSUE TITLE
More Character images

### ISSUE THREAD

**Issue by kingsmen732** [kingsmen732/portfolio#46]
More Character images generated in a general audience in mind so people from different races also can just change it and use them.
 
 basically create more options ☺️

**Comment by kingsmen732** [comment]
![image](https://github.com/user-attachments/assets/6c27c1db-f7db-4ece-80f3-fed9667cf929)
![image](https://github.com/user-attachments/assets/c13cd623-9959-45b9-b731-1c1a9e288ec0)
![image](https://github.com/user-attachments/assets/53455ea2-433d-4e10-b552-163ef6b16a04)

**Comment by NechamaSopher** [comment]
Hi, @kingsmen732 Do you mean to add more svg images?

**Comment by kingsmen732** [comment]
Yes more characters svg for options for user to select according to their perspectives

**Comment by NechamaSopher** [comment]
Greate, Can you assign this to me?
